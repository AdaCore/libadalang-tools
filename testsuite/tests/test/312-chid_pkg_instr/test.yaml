description: Test that gnattest correctly creates bodies for child units

driver: gnattest_tgen

default-gpr: true
