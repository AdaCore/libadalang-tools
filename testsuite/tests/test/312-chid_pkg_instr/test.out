pkg-child.ads:7:7: info: corresponding test PASSED
pkg-child.ads:7:7: info: corresponding test PASSED
pkg-child.ads:7:7: info: corresponding test PASSED
pkg-child.ads:7:7: info: corresponding test PASSED
pkg-child.ads:7:7: info: corresponding test PASSED
pkg-child.ads:13:7: info: corresponding test PASSED
pkg-child.ads:13:7: info: corresponding test PASSED
pkg-child.ads:13:7: info: corresponding test PASSED
pkg-child.ads:13:7: info: corresponding test PASSED
pkg-child.ads:13:7: info: corresponding test PASSED
pkg-child.ads:3:4: info: corresponding test PASSED
pkg-child.ads:3:4: info: corresponding test PASSED
pkg-child.ads:3:4: info: corresponding test PASSED
pkg-child.ads:3:4: info: corresponding test PASSED
pkg-child.ads:3:4: info: corresponding test PASSED
pkg-child.ads:17:4: info: corresponding test PASSED
pkg-child.ads:17:4: info: corresponding test PASSED
pkg-child.ads:17:4: info: corresponding test PASSED
pkg-child.ads:17:4: info: corresponding test PASSED
pkg-child.ads:17:4: info: corresponding test PASSED
pkg-child.ads:3:4: error: corresponding test FAILED: Test not implemented. (pkg-child-test_data-tests.adb:61)
pkg-child.ads:17:4: error: corresponding test FAILED: Test not implemented. (pkg-child-test_data-tests.adb:97)
pkg-child.ads:7:7: error: corresponding test FAILED: Test not implemented. (pkg-child-test_data-tests-inner_1-test_data-tests.adb:61)
pkg-child.ads:13:7: error: corresponding test FAILED: Test not implemented. (pkg-child-test_data-tests-inner_2-test_data-tests.adb:61)
24 tests run: 20 passed; 4 failed; 0 crashed.
Test runner dumped 20 tests
